content/actions/guides/deploying-to-google-kubernetes-engine.md

```diff
@@ -11,68 +11,89 @@ versions:
 {% data reusables.actions.enterprise-github-hosted-runners %}
 
 ### Introduction
-[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE) is a managed Kubernetes cluster service from Google Cloud and is a great option for hosting your containerized workloads in the cloud or on premise.
 
-This guide will show you how to use GitHub Actions to build and deploy a containerized application from Google Container Registry (GCR) to GKE. 
+This guide explains how to use {% data variables.product.prodname_actions %} to build a containerized application in Google Container Registry (GCR) and deploy it to Google Kubernetes Engine (GKE).
```

Comment:
@martin389, what would you think about something like this instead 💭...

> ...{% data variables.product.prodname_actions %} to build a containerized application, push it to Google Container Registry (GCR), and deploy it to Google Kubernetes Engine (GKE)

The build is taken care of by GitHub Actions, and GCR is used an intermediary registry for GKE to pull from.

Reply:
Sounds good to me, I'll make the change 👍